src/aggregator/integration/multi_server_forwarding_pipeline_test.go

```diff
@@ -131,14 +131,15 @@ func testMultiServerForwardingPipeline(t *testing.T, discardNaNAggregatedValues
 		}
 	}
 
+	clusterClient := memcluster.New(kv.NewOverrideOptions())
 	instances := make([]placement.Instance, 0, len(multiServerSetup))
 	for _, mss := range multiServerSetup {
 		instance := mss.instanceConfig.newPlacementInstance()
 		instances = append(instances, instance)
 	}
-	initPlacement := newPlacement(numTotalShards, instances)
-	require.NoError(t, setPlacement(placementKey, kvStore, initPlacement))
-	topicService, err := initializeTopic(defaultTopicName, kvStore, initPlacement)
+	initPlacement := newPlacement(numTotalShards, instances).SetReplicaFactor(2)
```

Comment:
What's the reason to `SetReplicaFactor(2)` with this change?

Reply:
After recent changes, placement updates now go through `PlacementService()`, which validates placement before making changes. Since this test defines replicated cluster, replication factor has to be set accordingly to pass validation.

https://github.com/m3db/m3/blob/abc3d63621ae21ef31f15cb66447fe72d559b235/src/aggregator/integration/multi_server_forwarding_pipeline_test.go#L79-L125